lms/envs/aws.py

```diff
@@ -999,3 +999,6 @@
 
 # Add an ICP license for serving content in China if your organization is registered to do so
 ICP_LICENSE = ENV_TOKENS.get('ICP_LICENSE', None)
+
+############## Settings for CourseGraph ############################
+COURSEGRAPH_JOB_QUEUE = LOW_PRIORITY_QUEUE
```

Comment:
If you define your queues [like this](https://github.com/edx/edx-platform/blob/69d76b69d7df6f89750adac77f20ee6fa9b8fa73/lms/envs/aws.py#L270) it becomes possible to change the queue assigned to this task on the fly with just a config change; I highly recommend unless we're sure this task will always want to be on the low queue.